src/cache.ts

```diff
@@ -1,14 +1,16 @@
-import avro from 'avsc'
+import avro, { ForSchemaOptions } from 'avsc'
 
 import { Schema } from './@types'
 
 export default class Cache {
   registryIdBySubject: { [key: string]: number }
   schemasByRegistryId: { [key: string]: Schema }
+  forSchemaOptions?: Partial<ForSchemaOptions>
 
-  constructor() {
+  constructor(forSchemaOptions?: Partial<ForSchemaOptions>) {
```

Comment:
This'd ideally also be an object

Reply:
Since `cache` is internal class,  I feel it's fine to keep this single argument,  ok?

Reply:
Since `cache` is internal class, I feel it's fine to keep this single argument, ok?

Reply:
Yes, it could easily be changed later 👍🏼